README.rst

```diff
@@ -171,8 +175,29 @@ For reasons that we don't understand, the Fortran compiler can give odd errors,
 This appears to be Windows' fault, rather than conda's (?).
 We highly recommend trying both the Anaconda prompt, as well as the standard dos prompt (press start: type `cmd`) if you experience errors related to compilation.
 
-If you want to build the documentation, all run: `conda install -y -c conda-forge sphinx_rtd_theme sphinx-gallery`
-
+If you want to build the documentation, run: `conda install -y -c conda-forge sphinx_rtd_theme sphinx-gallery`
+
+Setting up your _pyspecdata configuration file
+----------------------------------------------
+Part of the pySpecData package is the datadir module, allowing the user to run the same code on 
+different machines - even thought he location of the raw spectral data might change. 
+This is controlled by the ``~/.pyspecdata`` or ``~/_pyspecdata`` config file. 
+in the key ``[General]`` there should be a value pair for ``data_directory = ``.
+The value should correspond either to your Rclone remote or you can simply copy and
+paste the location of your data on the machine.
```

Comment:
I would give this as an example (use a [literal block](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks) to format)

```
[General]
data_directory = /home/jmfranck/exp_data
qesr conversion = 162.66
default q = 4700

[ExpTypes]
odnp_nmr_comp/odnp = /home/jmfranck/exp_data/NMR_comp/ODNP

[mode]
figures = standard

[RcloneRemotes]
nmr_comp/odnp = jmf_teams:General/exp_data/NMR_comp/ODNP/
```
I would use a bulleted list to explain the file in order.

Work in this phrase: "where `jmf_teams` is a properly configured (rclone)[https://rclone.org/] remote that shows up in response to the shell command `rclone config`"